src/lib/services/types/staking.ts

```diff
@@ -16,6 +24,54 @@ export const zStakingParamsResponseLcd = z
   })
   .transform(snakeToCamel);
 
+export const zDistributionParamsResponseLcd = z
+  .object({
+    params: z.object({
+      community_tax: zBig,
+      base_proposer_reward: zBig,
+      bonus_proposer_reward: zBig,
+      withdraw_addr_enabled: z.boolean(),
+    }),
+  })
+  .transform(snakeToCamel);
+
+export const zAnnualProvisionsResponseLcd = z
+  .object({
+    annual_provisions: zBig,
+  })
+  .transform(snakeToCamel);
+
+export const zMintParamsResponseLcd = z
+  .object({
+    params: z.object({
+      mint_denom: z.string(),
+      genesis_epoch_provisions: zBig,
+      epoch_identifier: z.string(),
+      reduction_period_in_epochs: zBig,
+      reduction_factor: zBig,
+      distribution_proportions: z.object({
+        staking: zBig,
+        pool_incentives: zBig,
+        developer_rewards: zBig,
+        community_pool: zBig,
+      }),
+      weighted_developer_rewards_receivers: z.array(
+        z.object({
+          address: zHexAddr20,
```

Comment:
@songwongtp Can it be the contract address?

Reply:
yep, and should be `zBechAddr` instead of hex too